CHANGELOG.rst

```diff
@@ -2,6 +2,12 @@
 CHANGELOG
 =========
 
+1.0.dev4
+=====
+
+* feature: LinearLearnerEstimator: Add more hyper-parameters
```

Comment:
Can you prefix this with "Spark / PySpark" to denote that it affects both libraries?